pdns/dnsrulactions.hh

```diff
@@ -126,7 +126,7 @@ public:
   }
   bool matches(const DNSQuestion* dq) const override
   {
-    return dq->dh->cd || (getEDNSZ((const char*)dq->dh, dq->len) & EDNS_HEADER_FLAG_DO);    // turns out dig sets ad by default..
+    return dq->dh->ad || (getEDNSZ((const char*)dq->dh, dq->len) & EDNS_HEADER_FLAG_DO);
```

Comment:
Hmm, this looks wrong, or at least would not be consistent with our documented behavior.